Compile fails with "package sun.dc.pr does not exist"

Running mvn compile (or mvn clean package) on master fails with

```
[INFO] Reactor Summary:
[INFO] 
[INFO] protege-parent ..................................... SUCCESS [  0.237 s]
[INFO] protege-launcher ................................... SUCCESS [  5.547 s]
[INFO] protege-common ..................................... SUCCESS [  2.225 s]
[INFO] protege-editor-core ................................ SUCCESS [  5.437 s]
[INFO] protege-editor-owl ................................. FAILURE [  3.823 s]
[INFO] protege-desktop .................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 17.790 s
[INFO] Finished at: 2016-10-24T19:40:23+01:00
[INFO] Final Memory: 41M/635M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.3:compile (default-compile) on project protege-editor-owl: Compilation failure
[ERROR] /home/mans0954/java/protege/protege-editor-owl/src/main/java/org/protege/editor/owl/ui/action/DeleteClassAction.java:[11,17] package sun.dc.pr does not exist
```

--- a/protege-editor-owl/src/main/java/org/protege/editor/owl/ui/action/DeleteClassAction.java
+++ b/protege-editor-owl/src/main/java/org/protege/editor/owl/ui/action/DeleteClassAction.java
@@ -8,7 +8,6 @@ import org.protege.editor.owl.ui.renderer.OWLEntityIcon;
 import org.protege.editor.owl.ui.renderer.OWLSystemColors;
 import org.semanticweb.owlapi.model.OWLClass;
 import org.semanticweb.owlapi.util.OWLEntitySetProvider;
-import sun.dc.pr.PRError;
 
 
 /**

